packages/react-ui-ag/src/Listings/MobileListing.js

```diff
@@ -5,7 +5,7 @@ import classnames from 'classnames'
 import { ListingComponents, Text } from '@rentpath/react-ui-core'
 import Listing from './Listing'
 
-@themed(/^MobileListing/, { pure: true })
+@themed(/^(MobileListing|PropertyLink$)/, { pure: true })
```

Comment:
why not just change theme key to theme.MobileListing_PropertyLink

Reply:
I guess we could do that instead. Makes more sense, I'll change it.